src/main/java/org/checkerframework/specimin/Slicer.java

```diff
@@ -182,7 +182,9 @@ private void handleElement(Node node) {
       try {
         // Resolve isn't perfect: methods/constructors, even if in the same file, will not resolve
         // if there are unresolvable argument types
-        resolved = asResolvable.resolve();
+        if (!(node instanceof FieldDeclaration)) {
+          resolved = asResolvable.resolve();
```

Comment:
Why is this call to `resolve` skipped if `node` is a `FieldDeclaration`? Adding an explanation to the comment would be helpful.